app/database/migrations/2019_10_04_023504_add_message_to_realms_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddMessageToRealmsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('realms', function (Blueprint $table) {
+            $table->text('message')->nullable();
```

Comment:
Maybe change column name to 'motd'? What do you think? I think it's more intentful

Reply:
Might also need a column motd_updated (datetime) to show when it was last changed

Reply:
Also please place it somewhere sensible, not at the end of the table. Before created_at should be good. With motd_updated after motd

Reply:
I think the motd stuff sounds good. I want sure what we were even going to call it, but message of the day with maybe an expiration would be great.